src/fights.ts

```diff
@@ -2193,6 +2193,16 @@ const itemStealZones = [
     openCost: () => 0,
     preReq: null,
   },
+  {
+    location: $location`Shadow Rift`,
+    monster: $monster`shadow slab`,
+    item: $item`shadow brick`,
+    requireMapTheMonsters: true,
```

Comment:
i swear when I tried to map the monsters in the rift, it just didnt work (i think because its nowander?)

Reply:
You're probably right. I've tried to address this but am out of my depth without some more familiarity on the item steal zone code